library/src/main/java/com/google/android/exoplayer2/text/Cue.java

```diff
@@ -134,10 +143,26 @@
   /**
    * The size of the cue box in the writing direction specified as a fraction of the viewport size
    * in that direction, or {@link #DIMEN_UNSET}.
+   * <p>
+   * If {@link #bitmap} is not null then this value is used to indicate the width
    */
   public final float size;
 
   /**
+   * The height size of the cue box when a {@link #bitmap} is set specified as a fraction of the
+   * viewport size in that direction, or {@link #DIMEN_UNSET}.
+   */
+  public final float size_height;
```

Comment:
What's this used for? We already have the width. Can't the height be inferred? I doubt it would make sense to allow the bitmaps to be deformed into another aspect ratio, which is the only case I can think of where this isn't redundant.